bpf/include/inner_map.h

```diff
@@ -0,0 +1,9 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
```

Comment:
As macro `KERNEL_VERSION_HIGHER_5_13_0` is removed from `kmesh_macros_def.h`, make macro `BPF_INNER_MAP_DATA_LEN as a standalone file to prevent including `bpf/include/common.h`(as macro `KERNEL_VERSION_HIGHER_5_13_0` is used in it)